frontend/src/app/modules/common/remote-field-updater/remote-field-updater.component.ts

```diff
@@ -29,10 +29,11 @@
 import {Component, ElementRef, OnInit} from '@angular/core';
 import {keyCodes} from 'core-app/modules/common/keyCodes.enum';
 import {HttpClient} from '@angular/common/http';
-import {DynamicBootstrapper} from 'core-app/globals/dynamic-bootstrapper';
+
+export const remoteFieldUpdaterSelector = 'remote-field-updater';
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/3b1eae5a64159f8a044579a25870c772186dbd3f/frontend/src/app/modules/common/remote-field-updater/remote-field-updater.component.ts#L34"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>